Tests/Extensions/DSLExtensions.swift

```diff
@@ -2,11 +2,23 @@ import Quick
 import Nimble
 import Difference
 
-public func they(_ description: String, flags: FilterFlags = [:], file: String = #file, line: UInt = #line, closure: @escaping () -> Void) {
+public func they(
+    _ description: String,
+    flags: FilterFlags = [:],
+    file: StaticString = #file,
```

Comment:
I believe all our uses of `#file` want to become `#filePath` now, per recent Swift change.

Reply:
good catch!